components/webinar/CountdownTimer.tsx

```
@@ -0,0 +1,188 @@
+"use client";
+
+import React, { useEffect, useState } from "react";
+
+import { type CountdownTimer } from "@/types";
+import { cn } from "@/lib/utils";
+import { WebinarStatusEnum } from "@prisma/client";
+import { changeWebinarStatus } from "@/actions/webinar.actions";
+
+const CountdownTimer = ({
```

Biome (2.1.2)

[error] Shouldn't redeclare 'CountdownTimer'. Consider to delete it or rename it.

'CountdownTimer' is defined here:


(lint/suspicious/noRedeclare)